Georgiy_Kobiakov/02.GIT.Local/02.GIT.Local.md

```diff
@@ -0,0 +1,85 @@
+## Task 1
+cd d:
```

Comment:
please use code block to markup history of your commands

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks